spec/requests/api/rest/admin/cdr/cdrs_spec.rb

```diff
@@ -20,5 +22,65 @@
     end
 
     it_behaves_like :json_api_admin_check_authorization
+
+    context 'with filters' do
+      let(:json_api_request_params) do
+        { filter: filter }
+      end
+
+      context 'by src_prefix_in_eq' do
+        let(:filter) { { src_prefix_in_eq: '111111111111' } }
+        let!(:record) { FactoryBot.create(:cdr, src_prefix_in: '111111111111') }
+
+        include_examples :returns_json_api_collection do
+          let(:json_api_collection_ids) { [record.id.to_s] }
+        end
+      end
+
+      context 'by src_prefix_in_contains' do
+        let(:filter) { { src_prefix_in_contains: '111111111111' } }
```

Comment:
since you are checking `_contains` filter here - use part of `record.src_prefix_in`

same for `src_prefix_routing_contains` `dst_prefix_routing_contains`